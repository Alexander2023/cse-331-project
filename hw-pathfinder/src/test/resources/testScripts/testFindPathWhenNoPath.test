# Tests that path finding works properly when there is no path between existing nodes

CreateGraph g

AddNode g p1

AddNode g p2

FindPath g p1 p2