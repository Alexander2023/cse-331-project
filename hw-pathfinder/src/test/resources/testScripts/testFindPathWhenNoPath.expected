# Tests that path finding works properly when there is no path between existing nodes

created graph g

added node p1 to g

added node p2 to g

path from p1 to p2:
no path found